.github/workflows/test.yml

```diff
@@ -40,8 +40,7 @@ jobs:
     env:
       # The "default" npm is the one that ships with a given version of node.
       # For more: https://nodejs.org/en/download/releases/
-      IS_MRT_NODE: ${{ matrix.node == 16 && matrix.npm == 8 }}
```

Comment:
`IS_MRT_NODE` was added intentionally to keep track of the node version supported by MRT. They are dropping `node14` at some point so we're assuming `node16` as the default, but unlike `IS_DEFAULT_NPM` the intention of this env var is to track **exactly one** combination of `npm` + `node` and make that the one we expect deployed customer instances to use. We in turn right lighthouse CI on **just that one** combination to avoid running lighthouse against many combinations. If your work here parallelizing is sufficiently effective so that testing many combinations doesn't incur much penalty in total runtime, then I'm open to that.

Reply:
`IS_MRT_NODE` is no longer used in this job. The new job only runs lighthouse once, for node 16 w/ default npm.

Reply:
@bfeister  The git diff is confusing, but basically all I'm doing here is to remove `IS_MRT_NODE` because it's not being used and I moved lighthouse to its own job, still running on Node 16.

Reply:
Yeah that's fine, I got it. Comment was not blocking